test_runner/regress/test_endpoint_crash.py

```diff
@@ -23,3 +24,16 @@ def test_endpoint_crash(neon_env_builder: NeonEnvBuilder, sql_func: str):
     endpoint.safe_psql("CREATE EXTENSION neon_test_utils;")
     with pytest.raises(Exception, match="This probably means the server terminated abnormally"):
         endpoint.safe_psql(f"SELECT {sql_func}();")
+
+
+@pytest.mark.skipif(not WITH_SANITIZERS, reason="Sanitizers are not enabled, skipping")
+def test_sanitizers(neon_env_builder: NeonEnvBuilder):
+    """
+    Test that undefined behavior leads to endpoint abort with sanitizers enabled
+    """
+    env = neon_env_builder.init_start()
+    env.create_branch("test_ubsan")
+    endpoint = env.endpoints.create_start("test_ubsan")
+
+    with pytest.raises(Exception, match="This probably means the server terminated abnormally"):
+        endpoint.safe_psql("SELECT 1::int4 << 128")  # See BUG #17167
```

Comment:
https://github.com/neondatabase/neon/issues/17167 is 404

Reply:
I meant Postgres bug 17167 (I used the same reference format as can be found in postgres sources): https://www.postgresql.org/message-id/17167-028026e4ca333817@postgresql.org
Should I specify the full URL there?

Reply:
Yes, please, let's use the full URL